src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/RoleMappings.java

```diff
@@ -0,0 +1,12 @@
+package com.amazon.opendistroforelasticsearch.security.securityconf;
+
+import java.util.Collections;
+import java.util.List;
+
+public abstract class RoleMappings {
```

Comment:
I don't think this class should be `abstract`.

Reply:
made it a normal class